pipelines.adoc

```diff
@@ -1,266 +1,189 @@
-these resources:
-https://kabanero.io/docs/ref/general/tekton-webhooks.html
-https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md
+You can link:https://kabanero.io/docs/ref/general/tekton-webhooks.html[use the Tekton Dashboard Webhook Extension to drive pipelines] that automatically build and deploy an application whenever you update the code in your Git repo. Events such as commits or pull requests can be set up to automatically trigger pipeline runs.
 
 === Manual pipeline execution using a script
 
-If you are developing a new pipeline and would like to manually test it
-in a tight loop, using the script or manually driving pipelines might be
-more helpful than using a webhook. Steps below describe how to do that.
+If you are developing a new pipeline and want to manually test it in a tight loop, you might want to use a script or manually drive the pipeline.
 
-Login to your cluster. For example for OKD,
+. Log in to your cluster. For example for OKD,
 
-....
+```
 oc login <master node IP>:8443
-....
+```
 
-==== Clone the kabanero pipelines repo
+. Clone the kabanero pipelines repo
 
-....
+```
 git clone https://github.com/kabanero-io/kabanero-pipelines
-....
+```
 
-==== Run the script with the appropriate parameters
+. Run the following script with the appropriate parameters
 
-....
+```
 cd ./pipelines/incubator/manual-pipeline-runs/
 
 ./manual-pipeline-run-script.sh -r [git_repo of the Appsody project] -i [docker registery path of the image to be created] -c [collections name of which pipeline to be run]"
-....
+```
 
 For example:
 
-....
+```
 ./manual-pipeline-run-script.sh -r https://github.com/mygitid/appsody-test-project -i index.docker.io/mydockeid/my-java-microprofile-image -c java-microprofile"
-....
+```
 
-=== Manual pipeline execution via command line
+=== Manual pipeline execution from the command line
 
-Login to your cluster. For example for OKD,
+. Login to your cluster. For example for OKD,
 
-....
+```
 oc login <master node IP>:8443
-....
+```
 
-==== Clone the kabanero pipelines repo
+. Clone the kabanero pipelines repo
 
-....
+```
 git clone https://github.com/kabanero-io/kabanero-pipelines
 cd kabanero-pipelines
-....
+```
 
-==== Create PipelineResources
+. Create Pipeline resources
 
-Update the pipeline-resources.yaml with github & docker repo info to
-create the PipelineResources. Sample pipeline-resources.yaml files are
-provided for each featured collection under the manual-pipeline-runs
-dir. Update the docker-image URL. You can use the sample github repo
-provided or update it to point to your github repo.
+Update the `pipeline-resources.yaml` file with your GitHub & Docker repo information to create the `PipelineResources`. Sample `pipeline-resources.yaml` files are provided for each featured collection in the `manual-pipeline-runs` directory. Update the docker-image URL. You can use the sample GitHub rep or update it to point to your own GitHub repo.
```

Comment:
should be repo instead of rep

Reply:
yes in this location: https://github.com/kabanero-io/kabanero-pipelines/tree/master/pipelines/incubator/manual-pipeline-runs